cmd/external-plugins/cherrypicker/server.go

```diff
@@ -299,66 +349,62 @@ func (s *Server) handlePullRequest(l *logrus.Entry, pre github.PullRequestEvent)
 	}}, comments...)
 
 	// first look for our special comments
-	for i := range comments {
-		c := comments[i]
-		cherryPickMatches := cherryPickRe.FindAllStringSubmatch(c.Body, -1)
-		for _, match := range cherryPickMatches {
-			targetBranch := strings.Fields(match[1])
-			if requestorToComments[c.User.Login] == nil {
-				requestorToComments[c.User.Login] = make(map[string]*github.IssueComment)
+	for _, comment := range comments {
+		commands := parseComent(comment)
+
+		for targetBranch, chainedBranches := range commands {
+			if requesterToComments[comment.User.Login] == nil {
+				requesterToComments[comment.User.Login] = make(map[string]*github.IssueComment)
 			}
-			requestorToComments[c.User.Login][targetBranch[0]] = &c
-			if len(targetBranch) > 1 {
-				targetBranchToChainBranches[targetBranch[0]] = targetBranch[1:]
+			requesterToComments[comment.User.Login][targetBranch] = &comment
```

Comment:
Linter job [complains about this](https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_prow/305/pull-prow-verify-lint/1847713726501752832#1:build-log.txt%3A195-196) but as of go 1.22 this should [no longer be an issue](https://go.dev/blog/loopvar-preview).

```
cmd/external-plugins/cherrypicker/server.go:359:61: exporting a pointer for the loop variable comment (exportloopref)
			requesterToComments[comment.User.Login][targetBranch] = &comment 
```

I think we can simply drop this line from our lint config:

https://github.com/kubernetes-sigs/prow/blob/9efd512d94f74ffc64fdce0ce92c175fc12de0c8/.golangci.yml#L26

Reply:
Yeah I was already working on another PR where that linter is replaced with copyloopvar. I think we can temporarily simply drop the exportloopref linter.

Reply:
Much appreciated! I imagine fixing all copy loopvars  occurrences will be quite a PR ;)